src/app/shared/login.service.ts

```diff
@@ -22,4 +30,18 @@ export class LoginService {
     window.location.href = this.githubUrl;
   }
 
+  public redirectAfterLogin() {
+    let redirectUrl = this.localStorage.get<string>(this.REDIRECT_URL_KEY);
```

Comment:
we should work out how to do this without local storage, maybe Dana can help with that.